internal/cmd/kafka/command_link_create.go

```diff
@@ -167,36 +193,124 @@ func (c *linkCommand) create(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+var sensitiveLinkConfigs = map[string]bool{
+	saslJaasConfigPropertyName:      true,
+	localSaslJaasConfigPropertyName: true,
+}
+
+func linkConfigsCommandOutput(linkConfig map[string]string) string {
+	filtered := make(map[string]string)
+	for key, val := range linkConfig {
+		if _, ok := sensitiveLinkConfigs[key]; ok {
+			filtered[key] = "***"
+		} else {
+			filtered[key] = val
+		}
+	}
+	return createKeyValuePairs(filtered)
+}
+
+func createKeyValuePairs(m map[string]string) string {
```

Comment:
This looks like the inverse of the `toMap()` function in internal/pkg/properties/properties.go. Can we define this function there and unit test it? It seems like we should be able to repurpose this in other areas of the CLI.

Reply:
I copied this function from a Stack Overflow answer - that's not good enough? ;) Aka will add some unit tests.

Reply:
Moved this method to internal/pkg/properties/properties.go and added some tests